### PR TITLE
[Manta] [Ready to merge] Integrate manta pay pallet

### PR DESCRIPTION
**Pull request by zhenfeizhang**
The current integration has been tested for both initialization and minting function.
The transfer and reclaim functions are to be tested -- requires some updates on the CLI.

### CODE REVIEW

File: runtimes/manta/runtime/Cargo.toml [hunk 1]
```diff
@@ -53,8 +53,14 @@ pallet-timestamp = { default-features = false, version = '3.0.0' }
 pallet-transaction-payment = { default-features = false, version = '3.0.0' }
 pallet-transaction-payment-rpc-runtime-api = { default-features = false, version = '3.0.0' }
 
+# Manta Pay
+# todo: update the branch dependency to calamari once this branch is merged
```
Comment (stechu) [inline runtimes/manta/runtime/Cargo.toml]: This branch has been merged now :)

**Review by stechu — CHANGES_REQUESTED** [review]
LGTM in general, two things left:
1. Update the `runtime/Cargo.toml` to `calamari` branch
2. should merge master after #18 is merged though.

File: runtimes/manta/runtime/Cargo.toml (hunk 1 above)
Comment (zhenfeizhang) [reply runtimes/manta/runtime/Cargo.toml]: done

**Comment by zhenfeizhang** [comment]
> LGTM in general, two things left:
> 
>     1. Update the `runtime/Cargo.toml` to `calamari` branch
> 
>     2. should merge master after #18 is merged though.

Done.

**Comment by stechu** [comment]
Thanks for the changes.

**Pull request merged**